examples/MNIST/mnist_model.py

```diff
@@ -13,9 +13,13 @@
 import mlflow.pytorch
 import pytorch_lightning as pl
 import torch
-from pytorch_lightning.overrides.data_parallel import (
-    LightningDistributedDataParallel,
-    LightningDataParallel,
+# from pytorch_lightning.overrides.data_parallel import (
```

Comment:
What is the reason for this change?

Reply:
@chauhang, both lightningdataparallel and dataparallel is removed in latest pytorch lightning and they suggested to directly use torch.nn.parallel.DistributedDataParallel and torch.nn.parallel.DataParallel.

Reference: https://github.com/PyTorchLightning/pytorch-lightning/pull/7510/files